airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1362 @@
+
+        return client.get_inspect_template(
+            name=name, retry=retry, timeout=timeout, metadata=metadata
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def get_job_trigger(self, name, retry=None, timeout=None, metadata=None):
+        """
+        Gets a job trigger.
+
+        :param name: Resource name of the project and the triggeredJob.
+        :type name: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: google.cloud.dlp_v2.types.JobTrigger
+        """
+
+        client = self.get_conn()
+
+        if not name:
+            raise AirflowException("Please provide the triggeredJob name.")
+
+        return client.get_job_trigger(
+            name=name, retry=retry, timeout=timeout, metadata=metadata
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def get_stored_info_type(self, name, retry=None, timeout=None, metadata=None):
+        """
+        Gets a stored infoType.
+
+        :param name: Resource name of the organization and storedInfoType to be read.
+        :type name: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: google.cloud.dlp_v2.types.StoredInfoType
+        """
+
+        client = self.get_conn()
+
+        if not name:
+            raise AirflowException("Please provide the storedInfoType name.")
+
+        return client.get_stored_info_type(
+            name=name, retry=retry, timeout=timeout, metadata=metadata
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def inspect_content(
+        self,
+        parent,
+        inspect_config=None,
+        item=None,
+        inspect_template_name=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        Finds potentially sensitive info in content. This method has limits on input size,
+        processing time, and output size.
+
+        :param parent: The parent resource name.
```

Comment:
It is worth adding the possibility of defining the project ID using the connection configuration.